pkgs/by-name/ow/owntone/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  stdenv,
+  autoPatchelfHook,
+  autoreconfHook,
+  fetchFromGitHub,
+
+  avahi,
+  curl,
+  bison,
+  ffmpeg,
+  flex,
+  gettext,
+  gnutls,
+  gperf,
+  json_c,
+  libconfuse,
+  libevent,
+  libgcrypt,
+  libgpg-error,
+  libplist,
+  libsodium,
+  libtool,
+  libunistring,
+  libwebsockets,
+  libxml2,
+  pkg-config,
+  protobufc,
+  sqlite,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  version = "28.11";
+  pname = "owntone";
+
+  src = fetchFromGitHub {
+    owner = "owntone";
+    repo = "owntone-server";
+    tag = "${finalAttrs.version}";
+    hash = "sha256-lUelBZy7kAJ9tkvVbmxfXHiANJvhrnQydZkWiuoYTNU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    autoreconfHook
+  ];
+
```

Comment:
could you add `passthru.updateScript = nix-update-script { };`? Makes maintaining easier.